site/pages/[...slug].js

```diff
@@ -1,41 +1,38 @@
-import MdxPage from '../components/MDX';
 import { allOtherPages } from 'contentlayer/generated';
-import { useMDXComponent } from 'next-contentlayer/hooks';
 
+import MdxPage from '../components/MDX';
+import siteConfig from "../config/siteConfig"
+
+export default function Page({ body, ...meta }) {
+  const frontMatter = {
+    ...meta,
+    date: meta.date === "Invalid Date" ? null : meta.date,
+    created: meta.created === "Invalid Date" ? null : meta.created
```

Comment:
I have moved the dates logic to contentlayer config, so these can be safely removed now.

Reply:
Removed.